featurebyte/query_graph/sql/interpreter/preview.py

```diff
@@ -448,6 +387,8 @@ def construct_sample_sql(
             Column to apply date range filtering on
         total_num_rows: int
             Total number of rows before sampling
+        sample_on_primary_table: bool
+            Whether to sample on primary table
```

Comment:
Is it more correct for this comment to be "Whether the graph is already sampled on primary table"? Since `_construct_sample_sql` will no longer perform sampling on primary table, right?